PWGCF/JCorran/Tasks/jEPDzeroFlowAnalysis.cxx

```diff
@@ -0,0 +1,212 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+/// \author junlee.kim@cern.ch
+/// \since Jul 2024
+
+#include <experimental/type_traits>
+#include <cmath>
+#include <array>
+#include <cstdlib>
+#include <chrono>
+#include <string>
+#include <vector>
+
+#include "TLorentzVector.h"
+#include "TRandom3.h"
+#include "TF1.h"
+#include "TVector2.h"
+#include "Math/Vector3D.h"
+#include "Math/Vector4D.h"
+#include "Math/GenVector/Boost.h"
+#include <TMath.h>
+
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/HistogramRegistry.h"
+#include "Framework/StepTHn.h"
+#include "Framework/O2DatabasePDGPlugin.h"
+#include "Framework/ASoAHelpers.h"
+#include "Framework/StaticFor.h"
+
+#include "Common/DataModel/Centrality.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "Common/DataModel/EventSelection.h"
+#include "Common/DataModel/Qvectors.h"
+
+#include "Common/Core/trackUtilities.h"
+#include "Common/Core/TrackSelection.h"
+#include "Common/Core/EventPlaneHelper.h"
+
+#include "CommonConstants/PhysicsConstants.h"
+
+#include "ReconstructionDataFormats/Track.h"
+
+#include "DataFormatsParameters/GRPObject.h"
+#include "DataFormatsParameters/GRPMagField.h"
+
+#include "CCDB/CcdbApi.h"
+#include "CCDB/BasicCCDBManager.h"
+
+#include "PWGCF/DataModel/CorrelationsDerived.h"
+
+using namespace std;
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+using namespace o2::soa;
+using namespace o2::constants::physics;
+
+struct jEPDzeroFlowAnalysis {
+  using MyCollisions = soa::Join<aod::Collisions, aod::EvSels, aod::CentFT0Ms, aod::CentFT0Cs, aod::Qvectors>;
```

Comment:
Are you planning to use University of Jyvaskyla framework for your analysis?
I that was not the case then the task should be place under the Flow directory